app/src/main/java/com/example/parkingproyect/mvp/presenter/ParkingPresenter.java

```diff
@@ -5,25 +5,25 @@
 
 public class ParkingPresenter {
 
-      private Parking parking;
-      private ParkingView parkingView;
+    private Parking parking;
+    private ParkingView parkingView;
 
-      public ParkingPresenter(Parking parking, ParkingView parkingView){
-             this.parking = parking;
-             this.parkingView = parkingView;
-      }
-
-      public void onSizeCreationButtonPressed(){
-          //Declare the string variable to make it more clear
-          String size = parkingView.getSize();
-          try {
-              parking.setParkingSize(size);
-              parkingView.showSizeMessage(String.valueOf(size));
-          }
-          catch(NumberFormatException ex){
-              parkingView.showInvalidError();
-          }
-
-      }
+    public ParkingPresenter(Parking parking, ParkingView parkingView) {
+        this.parking = parking;
+        this.parkingView = parkingView;
+    }
 
+    public void onSizeCreationButtonPressed() {
+        String size = parkingView.getSize();
+        try {
+            if (Integer.parseInt(size) <= 0) {
```

Comment:
To avoid parsing the integer twice, you could make `Parking.setParkingSize` throw a NumberFormatException when size <= 0 or an IllegalArgumentException exception if you really want to show different error messages.